CrowdStrike/falconjs
# Swagger update 2024-08-19

- Regenerate from latest swagger
- Make ASPM API naming consistent with other abbreviated API's (i.e. lowercase)
- Ensure package index.ts isn't overwritten with a generated copy
- Work around devices API returning `errors: null` instead of `[]`
